jina/serve/runtimes/gateway/grpc/gateway.py

```diff
@@ -96,16 +96,11 @@ async def setup_server(self):
         self.logger.debug(f'start server bound to {bind_addr}')
         await self.server.start()
 
-    async def teardown(self):
+    async def shutdown(self):
         """Free other resources allocated with the server, e.g, gateway object, ..."""
-        await super().teardown()
-        await self.health_servicer.enter_graceful_shutdown()
-
-    async def stop_server(self):
-        """
-        Stop GRPC server
-        """
+        await self.streamer.close()
```

Comment:
can't we put this repetitive line in base class so users don't have to worry about the streamer object ?

Reply:
Okey, I will have `GatewayRuntime` close it

Reply:
Done